src/Fmt_ast.ml

```diff
@@ -3406,7 +3393,14 @@ and fmt_module c ?epi keyword name xargs xbody colon xmty attributes =
                    | _ -> fmt "" )
                  $ Option.call ~f:epi ) ))
   in
-  (fmt_docstring_around ~loc c doc)
+  let single_line =
+    String.length name.txt * 3 < c.conf.margin
```

Comment:
`* 3` ? Maybe module_*_is_simple could also return the length of simple expressions, so that this can be more precise.